lib/fluent/plugin/out_rewrite_tag_filter.rb

```diff
@@ -59,7 +59,11 @@ def configure(conf)
     end
 
     unless @remove_tag_prefix.nil?
-      @remove_tag_prefix = /^#{Regexp.escape(@remove_tag_prefix)}\.?/
+      if @remove_tag_prefix.start_with?("/") and @remove_tag_prefix.end_with?("/")
```

Comment:
How about use single quote?
```suggestion
      if @remove_tag_prefix.start_with?('/') and @remove_tag_prefix.end_with?('/')
```